server/lib/src/valueset/session.rs

```diff
@@ -360,11 +378,16 @@ impl ValueSetT for ValueSetSession {
                     label: m.label.clone(),
                     expiry: m.expiry.map(|odt| {
                         debug_assert!(odt.offset() == time::UtcOffset::UTC);
-                        odt.format(time::Format::Rfc3339)
+                        #[allow(clippy::expect_used)]
+                        odt.format(&time::format_description::well_known::Rfc3339)
+                            .expect("Failed to format timestamp into RFC3339!")
                     }),
                     issued_at: {
                         debug_assert!(m.issued_at.offset() == time::UtcOffset::UTC);
-                        m.issued_at.format(time::Format::Rfc3339)
+                        #[allow(clippy::expect_used)]
+                        m.issued_at
+                            .format(&time::format_description::well_known::Rfc3339)
```

Comment:
Should we just make Rfc3339 a "pub use" in lib.rs preludo to save using it like this so much?

Reply:
Done, with use statements elsewhere.